src/subscribables/subscribable.js

```diff
@@ -75,6 +79,20 @@ var ko_subscribable_fn = {
                 ko.dependencyDetection.end(); // End suppressing dependency detection
             }
         }
+        // Notify subscribers to the ALL_EVENT event, but not if that was the event that was fired (prevent double events)
+        if (this.hasSubscriptionsForEvent(allEvent) && event != allEvent) {
```

Comment:
Why don't you use notifysubscribers here?
If(event !== all) 
   NotifySubscribers("all")

No need in manual loop, dep detection etc